modules/installing-oci-about-agent-based-installer.adoc

```diff
@@ -31,7 +33,9 @@ By running your {product-title} cluster on {oci}, you can access the following c
 
 * Block Volume storage, where you can configure scaling and auto-tuning settings for your storage volume, so that the Block Volume service automatically adjusts the performance level to optimize performance.
 
-* {ocvs}, where you can deploy a cluster in a public-cloud environment that operates on a VMware® vSphere software-defined data center (SDDC). You continue to retain full-administrative control over your VMware vSphere environment, but you can use {oci} services to improve your applications on flexible, scalable, and secure infrastructure.
+// Should I remove the reference below of ocvs? If so, what should the new phrasing be?
+
+* {ocvs-first}, where you can deploy a cluster in a public-cloud environment that operates on a VMware® vSphere software-defined data center (SDDC). You continue to retain full-administrative control over your VMware vSphere environment, but you can use {oci} services to improve your applications on flexible, scalable, and secure infrastructure.
```

Comment:
I would posit that all three of these are more of a "which" than a "where", but I would leave them alone for now for consistency and reconsider phrasing later